db/schema.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 20_221_026_083_005) do
+  create_table 'comments', force: :cascade do |t|
+    t.text 'text'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.integer 'posts_id', null: false
+    t.integer 'users_id', null: false
+    t.index ['posts_id'], name: 'index_comments_on_posts_id'
+    t.index ['users_id'], name: 'index_comments_on_users_id'
+  end
+
+  create_table 'likes', force: :cascade do |t|
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.integer 'posts_id', null: false
+    t.integer 'users_id', null: false
+    t.index ['posts_id'], name: 'index_likes_on_posts_id'
+    t.index ['users_id'], name: 'index_likes_on_users_id'
+  end
+
+  create_table 'posts', force: :cascade do |t|
+    t.string 'title'
+    t.text 'text'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.integer 'users_id', null: false
```

Comment:
- [ ] Kindly add `:comments_counter :likes_counter` to your posts as well.

Reply:
comments_counter and likes_counter added to the posts table.